WPMedia/muckrock
# fix: Get autoimports working again

## Description
The `autoimport` task used a lot of S3 logic, but I didn't yet know how to test it when initially doing the `boto2` -> `boto3` migration. As a result, there were a couple of little errors I missed when rewriting the S3 code, and needed to fix before we can merge it into Muckrock upstream.

This PR fixes those little bugs, and allows auto imports to work again. It also makes them more configurable, via the `AWS_AUTOIMPORT_BUCKET_NAME` and `AWS_AUTOIMPORT_PATH` variables.

## Test Steps
1. Start Muckrock and Squarelet
2. Pick a request you want to upload a scanned attachment to, noting the `Muckrock#` on the left side of the page.
3. Upload any PDF file to `s3://wp-muckrock-uploads-dev/autoimport_scans/05-14-2021 MR#.pdf`, where `#` is the MR number you noted above
4. Go to the [Foia Request](http://dev.muckrock.com/admin/foia/foiarequest/) page in the admin, and click "Autoimport"
5. Check your local mail at http://dev.mailhog.com, where there should be a log from the auto import with a success message
6. Return to your chosen request, and you should see a new communication with your file attached to it. Click into it to make sure the `Download` link works as well.

## Screenshots
<img width="636" alt="Screen Shot 2021-05-14 at 10 54 08 AM" src="https://user-images.githubusercontent.com/3749412/118288793-b76ce200-b4a2-11eb-80db-e20dc204f777.png">
<img width="357" alt="Screen Shot 2021-05-14 at 10 54 19 AM" src="https://user-images.githubusercontent.com/3749412/118288827-bdfb5980-b4a2-11eb-85f9-10c2f2f6e99d.png">
